lib/baseClasses/HttpError.js

```diff
@@ -61,12 +66,6 @@ function HttpError() {
         code: self.name,
         message: self.message || ''
     };
-
-    // capture the stack trace because that doesn't happen automatically.
-    // the second parameter into captureStackTrace() is used to filter out
-    // parts of the stack trace that aren't relevant to the users.
-    // i.e., the error constructors shouldn't show up in the stack trace.
-    Error.captureStackTrace(self, options.constructorOpt || self.constructor);
```

Comment:
I must have missed this somewhere, but why aren't we capturing the stack trace anymore?